src/core/Microsoft.PowerToys.Settings.UI/Controls/HotkeySettingsControl.xaml

```diff
@@ -10,10 +10,10 @@
     d:DesignWidth="400">
     <StackPanel Orientation="Vertical">
         <ToolTipService.ToolTip>
-            <ToolTip>
+            <ToolTip Name="HotkeyTooltip">
                 <StackPanel Orientation="Vertical">
                     <TextBlock x:Uid="ShortcutWarningLabel"/>
-                    <TextBlock Text="{x:Bind Keys, Mode=OneTime}" FontWeight="SemiBold"/>
+                    <TextBlock Text="{x:Bind _keys, Mode=OneTime}" FontWeight="SemiBold"/>

```

Comment:
As I said on the comment above, I will try to remove that private field.

Reply:
I tried not creating the private property and had a problem with infinite loops, just push a PR that uses the public field instead of the private one!